pkg/solana/client/multinode_wrapped_client.go

```diff
@@ -0,0 +1,168 @@
+package client
+
+import (
+	"context"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/gagliardetto/solana-go/rpc"
+
+	mn "github.com/smartcontractkit/chainlink-solana/pkg/solana/client/multinode"
+)
+
+var _ ReaderWriter = (*MultiClient)(nil)
+
+// MultiClient - wrapper over multiple RPCs, underlying provider can be MultiNode or LazyLoader.
+// Main purpose is to eliminate need for frequent error handling on selection of a client.
+type MultiClient struct {
+	getClient func() (ReaderWriter, error)
+}
+
+func NewMultiNodeWrappedClient(getClient func() (ReaderWriter, error)) *MultiClient {
```

Comment:
```suggestion
func NewMultiClient(getClient func() (ReaderWriter, error)) *MultiClient {
```

Reply:
Should we maybe rename the file to `multiclient.go` since it's not only wrapping MultiNode?

Reply:
Good point. Addressed